Okta.Auth.Sdk/Okta.Auth.Sdk.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard1.3</TargetFramework>
-    <Version>1.0.1-beta</Version>
+    <Version>1.0.0</Version>
```

Comment:
Is Nuget okay with going from 1.0.1-beta to 1.0.0?  Will people currently on 1.0.1-beta get this as an update if they have `Include Prereleases` enabled?  (which they would have to have enabled if they were using the beta in the first place)

Probably this isn't a problem, but I don't know how it works so just wanted to hear your thoughts.

Reply:
Good catch here 💯 ! I double checked and this is how this works:
1.1.0-alpha < 1.1.0-alpha.1 < 1.1.0 < 1.1.1-alpha.1 < 1.1.1.
I'm going to change the version number to `1.0.1`.